shell/package.json

```diff
@@ -122,9 +122,12 @@
     "v-tooltip": "2.0.3",
     "vue": "2.7.16",
     "clipboard-polyfill": "4.0.1",
+    "ufo": "0.7.11",
```

Comment:
were ufo, unfetch and vue-router the deps that were brought in via nuxt, but given nuxt has been removed we have to manually add?

Reply:
Exactly 👍